src/App.web.tsx

```diff
@@ -18,28 +22,36 @@ const NAVIGATOR_SCREEN_OPTIONS = {
   },
 }
 
+const safeAreaStyles = { height: '100%' }
```

Comment:
Pour faire plus facilement la différence avec le style généré dynamiquement : 
```suggestion
const SAFE_AREA_STYLE = { height: '100%' }
```